docs/mobile/platform-specific-code.md

```diff
@@ -0,0 +1,15 @@
+## Mobile Playbook: Platform-specific code
+
+#### Prefer to separate `*.ios.js` and `*.android.js` component files over switching on `Platform.*`
```

Comment:
Sure. Then in this case, the `configurations/es6-react-native.js` should reflect the mandatory conventions differently than the rest of our files.